examples/demo-app/src/analytics.js

```diff
@@ -0,0 +1,53 @@
+import { ActionTypes } from "kepler.gl/actions";
+
+import { LOCATION_CHANGE } from "react-router-redux";
+
+const {
+  ADD_DATA,
+  LAYER_CONFIG_CHANGE,
+  LAYER_HOVER,
+  MAP_STYLE_CHANGE,
+  UPDATE_MAP,
+  TOGGLE_MODAL,
+  TOGGLE_SIDE_PANEL
+} = ActionTypes;
+
+const getPayload = ({ payload }) => payload;
+
+const trackingInformation = {
+  [ADD_DATA]: ({ payload: { files } }) =>
+    files.map(({ name, size, type }) => ({ name, size, type })),
```

Comment:
let's not track the name, only `type` and `size` would be sufficient

Reply:
Right, better not to upload anything about the users' data that they might not want